eng/config/PublishData.json

```diff
@@ -207,15 +207,15 @@
       "nugetKind": [ "Shipping", "NonShipping" ],
       "version": "3.9.*",
       "packageFeeds": "default",
-      "channels": [ "dev16.9p1" ],
+      "channels": [ "dev16.9p2" ],
       "vsBranch": "rel/d16.9",
       "vsMajorVersion": 16
     },
     "master-vs-deps": {
       "nugetKind": ["Shipping", "NonShipping"],
       "version": "3.8.*",
```

Comment:
Predates this PR but this should be 3.9.* too, right @JoeRobich ?

Reply:
Good eye. Now I wonder how we use that property since nothing blew up...

Reply:
Updated! Thanks for the catch.

Reply:
Looking at the code, I actually cannot see it being used anywhere...
I see no references to the branch's version from the $data object anywhere- https://github.com/dotnet/roslyn/blob/d7048e7263cc33c888d3f46e6097ede64085199b/eng/publish-assets.ps1